edx/edx-app-android
# Firebase Remote Configurations Integration for A/A Test

### Description

[LEARNER-7386](https://openedx.atlassian.net/browse/LEARNER-7386)
[LEARNER-7394](https://openedx.atlassian.net/browse/LEARNER-7394)
- Integrate Firebase Remote Configurations for A/A test.
- Trigger A/A test Event from course outline screen.

**Acceptance Criteria:**
- Build on the work done in the spike.
- The default value for the parameter should be “Not set”.
- Emit an event when the user views the course outline screen with different properties depending on the value of the remote parameter.
``` java
String aaExperimentGroup = mFirebaseRemoteConfig.getString("aa_experiment");
Analytics.with(context).track("Mobile Experiment Evaluated",
    new Properties().putValue("experiment", "aa_experiment")
                    .putValue("group", aaExperimentGroup)
                    .putValue("user_id", PUT_EDX_USER_ID_HERE)
                    .putValue("course_id", PUT_EDX_COURSE_ID_HERE)
);
```
**How to test this PR**
- Open a course from the Main dashboard screen.
- Check the analytics on the Segment named "**Mobile Experiment Evaluated**"

**Note:** Make sure Firebase configuration is enabled.
